uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/apis/leave-request-api.spec.js

```diff
@@ -187,6 +187,27 @@ define([
       });
     });
 
+    describe('getBalanceChangeBreakdown()', function () {
+      describe('basic tests', function () {
+        var leaveRequestId = 1;
+
+        beforeEach(function () {
+          spyOn(LeaveRequestAPI, 'sendGET').and.callThrough();
+          LeaveRequestAPI.getBalanceChangeBreakdown(leaveRequestId);
+          $rootScope.$digest();
+        });
+
+        afterEach(function () {
+          $httpBackend.flush();
+        });
+
+        it('calls the LeaveRequest.getBalanceChangeBreakdown endpoint', function () {
+          expect(LeaveRequestAPI.sendGET).toHaveBeenCalledWith(
+            'LeaveRequest', 'getBreakdown', { leave_request_id: leaveRequestId }, false);
+        });
```

Comment:
A test for the response is missing.

Reply:
I actually removed it by purpose, I don't really understand that logic which is current;y used in the test file:

1) Mock http request - OK
2) Check that a correct HTTP endpoint was called via sendGet - OK
3) Check that the response matches the mock - ? what does it actually test, how sendGet works?

Reply:
it could test that you are returning the exact value without any modifications. This can avoid adding changes to the api response when these changes should be reflected on the model.